include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -8573,6 +8573,126 @@
       ],
       "capabilities" : [ "SplitBarrierINTEL" ],
       "version" : "None"
+    },
+    {
+      "opname" : "OpGroupIMulKHR",
+      "class"  : "Group",
+      "opcode" : 6401,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdScope",        "name" : "'Execution'" },
+        { "kind" : "GroupOperation", "name" : "'Operation'" },
+        { "kind" : "IdRef",          "name" : "'X'" }
+      ],
+      "capabilities" : [ "GroupUniformArithmeticKHR" ],
+      "extensions" : [ "SPV_KHR_uniform_group_instructions" ],
```

Comment:
Can you please only include the "extensions" tag on the capability, and not on the instructions?  Rationale being that if the capability is gated by the extension, any instruction that depends on the capability will also be implicitly gated by the extension.

This change will apply to this instruction and all of the instructions below added in this PR.

Reply:
Thanks, done https://github.com/KhronosGroup/SPIRV-Headers/pull/269/commits/7744288e2d4a9c949ed0b6588a603c963551e025